frontend/src/dashboard.js

```diff
@@ -164,11 +164,6 @@ export class Dashboard extends React.Component {
         'selectedDashboard': targetObject.defaultDashboard,
         'dashboardInputValue': targetObject.defaultDashboard?.title
       })
-    } else {
```

Comment:
This is only one part of it - there are other dashboard selection bugs with widget fetching of the default dashboard